build.yml

```
@@ -49,48 +46,80 @@ pr:
 variables:
   skipComponentGovernanceDetection: true
 
-steps:
-  - task: ComponentGovernanceComponentDetection@0
-    inputs:
-      scanType: "Register"
-      verbosity: "Verbose"
-      alertWarningLevel: "High"
-
-  - task: UseNode@1
-    displayName: Use Node 20.x
-    inputs:
-      version: 20.x
+resources:
+  repositories:
+    - repository: m365Pipelines
+      type: git
+      name: 1ESPipelineTemplates/M365GPT
+      ref: refs/tags/release
 
-  # Install, build, lint, and test each package
-  - ${{ each parameter in parameters }}:
-      - task: Npm@1
-        displayName: Install - ${{ parameter.Value }}
-        inputs:
-          command: "custom"
-          workingDir: "${{ parameter.Value }}"
-          customCommand: "ci"
-      - task: CmdLine@2
-        displayName: Build - ${{ parameter.Value }}
-        inputs:
-          script: "npm run build"
-          workingDirectory: ${{ parameter.Value }}
-      - task: CmdLine@2
-        displayName: Lint - ${{ parameter.Value }}
-        inputs:
-          script: "npm run lint"
-          workingDirectory: ${{ parameter.Value }}
-      - task: CmdLine@2
-        displayName: Test - ${{ parameter.Value }}
-        inputs:
-          script: "npm run ci:test"
-          workingDirectory: ${{ parameter.Value }}
-      - task: PublishTestResults@2
-        displayName: Publish Test Results - ${{ parameter.Value }}
-        inputs:
-          testResultsFormat: "JUnit"
-          testResultsFiles: "**/*junit-report.xml"
-          searchFolder: ${{ parameter.Value }}/nyc
-          mergeTestResults: true
-        # Note while this runs if anything prior fails, the prior test steps won't
-        # run after first failure. Could be split into jobs to avoid interference.
-        condition: succeededOrFailed()
+extends:
+  template: v1/M365.Official.PipelineTemplate.yml@m365Pipelines
+  parameters:
+    pool:
+      name: NewLarge-linux-1ES
```

Don't think it needs the large pool
```suggestion
      name: Small-1ES
```